src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,98 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { addComments } from '../../api/comments';
 import './NewCommentForm.scss';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+type Props = {
+  postId: number | undefined,
+  addNewComment: (newComment: Comments) => void,
+};
+
+export const NewCommentForm: React.FC<Props> = ({ postId, addNewComment }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
+
+  const inputHandler = (event: React.ChangeEvent<HTMLInputElement
+  | HTMLTextAreaElement>) => {
```

Comment:
```suggestion
  const inputHandler = (
    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
  ) => {
```
also use `import { ChangeEvent } from 'react'` to make it shorter